src/ui/components/Library/Team/View/TestRuns/Overview/TestResultListItem.tsx

```diff
@@ -65,15 +66,22 @@ export function TestResultListItem({
     >
       <div className="flex flex-row items-center gap-1">
         {secondaryBadgeCount != null && <Icon className="ml-2 h-4 w-4" type="arrow-nested" />}
-        <div className=" flex cursor-pointer items-center justify-center transition">
-          <MaterialIcon iconSize="2xl" outlined style={{ color }}>
-            {["passed", "flaky"].includes(label) ? "play_circle" : "play_circle_filled"}
-          </MaterialIcon>
+        <div className="flex cursor-pointer items-center justify-center transition ">
+          <motion.div
+            className="m-auto h-6 w-6 rounded-full hover:cursor-pointer"
+            whileHover={{ scale: 1.1 }}
+            whileTap={{ scale: 1.0, boxShadow: "0px 0px 1px rgba(0,0,0,0.2)" }}
+            transition={{ duration: 0.05 }}
+          >
+            <MaterialIcon iconSize="2xl" outlined style={{ color }}>
+              {["passed", "flaky"].includes(label) ? "play_circle" : "play_circle_filled"}
+            </MaterialIcon>
+          </motion.div>
         </div>
       </div>
       <div className="flex shrink grow flex-col truncate">
         <div className="block truncate">{title || "Test"}</div>
-        <div className="truncate truncate text-xs text-bodySubColor">{filePath}</div>
+        <div className="truncate text-xs text-bodySubColor">{filePath}</div>
```

Comment:
Can we re-add the truncate style? I think keeping the rows uniform height goes a long way to making the page flow well.

Reply:
I reading on a small screen from bed so it's likely I'm misreading.

But I think truncate is still there? I think the diff shows it was written twice?

Reply:
Ugh, sorry my bad. The string "truncate" _was_ deleted but only because there were two of them.